public/js/showoff.js

```diff
@@ -1130,57 +1129,54 @@ function togglePreShow() {
 }
 
 function setupPreShow(seconds) {
-  if (preshow_secondsLeft = seconds) {
-    toggleFooter();
-    $.getJSON("preshow_files", false, function(data) {
-      $('#preso').after("<div id='preshow'></div><div id='tips'></div><div id='preshow_timer'></div>");
-      $.each(data, function(i, n) {
-        if(n == "preshow.json") {
-          // has a descriptions file
-          $.getJSON("/file/_preshow/preshow.json", false, function(data) {
-            preshow_des = data;
-          })
-        } else {
-          $('#preshow').append('<img ref="' + n + '" src="/file/_preshow/' + n + '"/>');
-        }
-      })
-      startPreShow();
+  preshow_stop = secondsFromNow(seconds);
+  try { presenterView.preshow_stop = preshow_stop } catch (e) {}
+
+  // footer styling looks icky. Hide it for now.
+  $('#footer').hide();
+
+  $.getJSON("preshow_files", false, function(data) {
+    $('#preso').after("<div id='preshow'></div><div id='tips'></div><div id='preshow_timer'></div>");
+    $.each(data, function(i, n) {
+      if(n == "preshow.json") {
+        // has a descriptions file
+        $.getJSON("/file/_preshow/preshow.json", false, function(data) {
+          preshow_des = data;
+        })
+      } else {
+        $('#preshow').append('<img ref="' + n + '" src="/file/_preshow/' + n + '"/>');
```

Comment:
This tries to load the README in _preshow as an img.  We can use JS to check file extension before this step or, hopefully, there is a better solution...